.gitignore

```diff
@@ -7,3 +7,5 @@ vulture.egg-info/
 .cache/
 .coverage
 .tox/
+.pytest_cache/
```

Comment:
Which command in our tests produces this directory? I can't generate this with "tox".

Reply:
I'm not sure about what command generates it, but yeah every time I run `tox`, it is there. 😛

Reply:
OK, my tox version was older. Now I see it. This should indeed be added to .gitignore. Can you remove the *.swp ignore, please?

Reply:
Done!